### ISSUE TITLE
Advanced Rocketry conflict with Littletiles causing crash when opening guidance computer (MP)

### ISSUE THREAD

**Issue by Hachuurui** [FTBTeam/FTB-Interactions#934]
## Modpack Version: 
2.0
## Bug Report:
When you try to access the guidance computer on any rocket you build, you'll crash on MP with an error stating the entityPlayerSP cannot be cast as entityPlayerMP

## Expected Behavior/Suggestions:
Shouldn't crash the client.

## Is It Repeatable? Steps to Reproduce: 
1. Build a rocket on multiplayer.
2. Attempt to access guidance computer on said rocket.
3. coin toss to see if client crashes.

## Link to Log or Crash File Paste**: 
https://paste.feed-the-beast.com/view/f67e29dc

## Mod/s Affected: 
Advanced Rocketry, Vulpes, Littletiles.

## Known Fix: 
Disassemble rocket then pull chip out or insert chip. If you interact with the rocket while its assembled, you will normally crash.

**Comment by Hachuurui** [comment]
Note: no little tile blocks are actually used on the rocket. Its pure Adv Rocketry, yet still throws the error from little tiles.

**Comment by Saereth** [comment]
I have opened an issue with littletile's dev, hopefully they can get a fix in for it
https://github.com/CreativeMD/LittleTiles/issues/438

**Comment by Saereth** [comment]
LT is planning a release for this in the next week or two 
https://github.com/CreativeMD/CreativeCore/issues/75

**Comment by mellamokb** [comment]
What ended up being the solution? I keep crashing every time I try to open the rocketry computer on the latest release, and I can't find the divine combination for it to work.

1. Tried upgrading LittleTiles/CreativeCore to latest version as of today.  Still crash
2. Tried removing LittleTiles/CreativeCore completely.  Still crash
3. Tried removing all extra mods I had added (LagGoggles, OptiFine).  Still crash

At this point the crash can't even identify the mod anymore and just says "Unknown" with no stacktrace, but it's still a ClassCastException.
